{
  "uid" : "6f60e2ba8d8471c3",
  "name" : "Successfully validating product by ID in the URL",
  "fullName" : "Integration-Tests.Produtos.TestProdutos.Successfully validating product by ID in the URL",
  "historyId" : "8676c9d8b23ef7a401994b8e38b1765e",
  "time" : {
    "start" : 1655220756777,
    "stop" : 1655220757391,
    "duration" : 614
  },
  "status" : "failed",
  "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
  "flaky" : true,
  "newFailed" : true,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
    "steps" : [ {
      "name" : "ProdutoKW.Search Products By Id In The URL",
      "time" : {
        "start" : 1655220756777,
        "stop" : 1655220757389,
        "duration" : 612
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ {
        "name" : "BuiltIn.Create Dictionary",
        "time" : {
          "start" : 1655220756778,
          "stop" : 1655220756778,
          "duration" : 0
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "825e0b6f40dbf7ea",
          "name" : "Keyword Log",
          "source" : "825e0b6f40dbf7ea.html",
          "type" : "text/html",
          "size" : 75
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "content-type=application/json"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.Create Session",
        "time" : {
          "start" : 1655220756779,
          "stop" : 1655220756779,
          "duration" : 0
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "5951d67e6b88975d",
          "name" : "Keyword Log",
          "source" : "5951d67e6b88975d.html",
          "type" : "text/html",
          "size" : 219
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "${BASE_URL}"
        }, {
          "name" : "arg3",
          "value" : "verify=True"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.GET On Session",
        "time" : {
          "start" : 1655220756780,
          "stop" : 1655220757387,
          "duration" : 607
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "ef273f05b67948a7",
          "name" : "Keyword Log",
          "source" : "ef273f05b67948a7.html",
          "type" : "text/html",
          "size" : 1296
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "/produtos/${product_id}"
        }, {
          "name" : "arg3",
          "value" : "headers=${headers}"
        }, {
          "name" : "arg4",
          "value" : "expected_status=any"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Log",
        "time" : {
          "start" : 1655220757388,
          "stop" : 1655220757388,
          "duration" : 0
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Set Test Variable",
        "time" : {
          "start" : 1655220757388,
          "stop" : 1655220757389,
          "duration" : 1
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "BeeJh5lz3k6kSIzA"
      } ],
      "stepsCount" : 5,
      "attachmentsCount" : 3,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "BuiltIn.Should Be Equal As Strings",
      "time" : {
        "start" : 1655220757390,
        "stop" : 1655220757390,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.status_code}"
      }, {
        "name" : "arg2",
        "value" : "200"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "BuiltIn.Should Be Equal As Strings",
      "time" : {
        "start" : 1655220757390,
        "stop" : 1655220757390,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f25441700>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.json()[\"nome\"]}"
      }, {
        "name" : "arg2",
        "value" : "Logitech MX Vertical"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Integration-Tests"
  }, {
    "name" : "suite",
    "value" : "Produtos"
  }, {
    "name" : "subSuite",
    "value" : "TestProdutos"
  }, {
    "name" : "framework",
    "value" : "robotframework"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "host",
    "value" : "fv-az180-72"
  }, {
    "name" : "thread",
    "value" : "default-bcd441f3-51a0-4588-8b56-57009bb64496"
  }, {
    "name" : "tag",
    "value" : "@integration_tests"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "tag",
    "value" : "@integration_tests"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 4,
        "unknown" : 0,
        "total" : 7
      },
      "items" : [ {
        "uid" : "baee3893a79f5cde",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/9//#testresult/baee3893a79f5cde",
        "status" : "passed",
        "time" : {
          "start" : 1655219662419,
          "stop" : 1655219662516,
          "duration" : 97
        }
      }, {
        "uid" : "d06d295a4b14630c",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/8//#testresult/d06d295a4b14630c",
        "status" : "passed",
        "time" : {
          "start" : 1655218423218,
          "stop" : 1655218423278,
          "duration" : 60
        }
      }, {
        "uid" : "92f27ceb135829b1",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/7//#testresult/92f27ceb135829b1",
        "status" : "passed",
        "time" : {
          "start" : 1655217744329,
          "stop" : 1655217744427,
          "duration" : 98
        }
      }, {
        "uid" : "24aaba1c64c93f00",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/5//#testresult/24aaba1c64c93f00",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217410016,
          "stop" : 1655217410632,
          "duration" : 616
        }
      }, {
        "uid" : "e64e533b96e63f2e",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/4//#testresult/e64e533b96e63f2e",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzA (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217115545,
          "stop" : 1655217116162,
          "duration" : 617
        }
      }, {
        "uid" : "8bb9f463a0b8a7c3",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/3//#testresult/8bb9f463a0b8a7c3",
        "status" : "passed",
        "time" : {
          "start" : 1655216713202,
          "stop" : 1655216713222,
          "duration" : 20
        }
      } ]
    },
    "tags" : [ "@regression", "@integration_tests", "@produtos" ]
  },
  "source" : "6f60e2ba8d8471c3.json",
  "parameterValues" : [ ]
}